README.md

```diff
@@ -88,6 +88,16 @@ The following test will run automatically as you try to pull request: -Lighthous
 
```

Comment:
- [ ] OPTIONAL
Live demo link on _line 49_ of your ReadMe is broken, kindly endeavor to fix it.

Reply:
I will gladly update the status once I get to the part of deploying the page.
In the mean time I have put a message of  "coming soon".

Reply:
@HunterX-7 
I think you can comment it out at meantime, till you're ready. 

Cheers! 😃